backend/webserver/models.py

```diff
@@ -15,3 +16,61 @@ class FollowRequest(models.Model):
 class Follow(models.Model):
     follower = models.ForeignKey(Author, related_name='following_authors', on_delete=models.CASCADE)
     followee = models.ForeignKey(Author, related_name='followed_by_authors', on_delete=models.CASCADE)
+
+class Post(models.Model):
+
+    author = models.ForeignKey(Author,on_delete=models.CASCADE)
+
+    created_at = models.DateTimeField(verbose_name="date created",null=False, blank=False, default = datetime.now)
+
+    edited_at = models.DateTimeField("date edited")
+
+    title = models.CharField(max_length=200)
+
+    description = models.CharField(max_length=200)
+
+    source = models.CharField(max_length=200)
+
+    origin = models.CharField(max_length=200)
+
+    unlisted = models.BooleanField(default=False) 
+
+    VISIBILITY_CHOICES = [
+        ("PUBLIC","Public"),
+        ("FRIENDS","Friends"),
+    ]
+
+    visibility = models.CharField(max_length=200,choices=VISIBILITY_CHOICES,default="PUBLIC")
+
+    CONTENT_TYPE_CHOICES = [
+        ("text/plain","Plain text"),
+        ("text/markdown","Markdown text")
+    ]
+
+    content_type = models.CharField(max_length=200,choices=CONTENT_TYPE_CHOICES,null=False,default="text/plain")
+
+    content = models.CharField(max_length=200)
+
+class Comment(models.Model):
+    author = models.ForeignKey(Author,on_delete=models.CASCADE)
+
+    post = models.ForeignKey(Post,on_delete=models.CASCADE)
+
+    content = models.CharField(max_length=200)
+
+    created_at = models.DateTimeField("created_at")
+
+    CONTENT_TYPE_CHOICES = [
+        ("text/plain","Plain text"),
+        ("text/markdown","Markdown text")
+    ]
+
+    content_type = models.CharField(max_length=200,choices=CONTENT_TYPE_CHOICES,null=False, default="text/plain")
+
+
+class Like(models.Model):
+    author = models.ForeignKey(Author,on_delete=models.CASCADE)
+
+    post = models.ForeignKey(Post,on_delete=models.CASCADE)
+
+    comment = models.ForeignKey(Comment,on_delete=models.CASCADE)
```

Comment:
```suggestion
    comment = models.ForeignKey(Comment, on_delete=models.CASCADE, null=True)
```
- It could either be a comment like or a post like